bazel/emscripten_deps.bzl

```diff
@@ -54,12 +54,17 @@ filegroup(
         "bin/clang{bin_extension}",
         "bin/llc{bin_extension}",
         "bin/llvm-ar{bin_extension}",
+        "bin/llvm-dwarfdump{bin_extension}",
+        "bin/llvm-mc{bin_extension}",
```

Comment:
I had to remove this line to get it to work on an apple silicon mac. The error was 
```
ERROR: /Users/greg/dev/repo/BUILD.bazel:39:10: Linking app failed: missing input file '@emscripten_bin_mac_arm64//:bin/llvm-mc'
```

Reply:
I think that file was removed from the prebuilt binaries.   It should not be needed by emscripten.

Reply:
Done